korman/operators/op_toolbox.py

```diff
@@ -174,33 +174,33 @@ def execute(self, context):
 
         
 class PlasmaToggleDoubleSidedOperator(ToolboxOperator, bpy.types.Operator):
-    bl_idname = "mesh.plasma_toggle_double_sided"
+    bl_idname = "mat.plasma_toggle_double_sided"
```

Comment:
Blender uses [`bpy.ops.material`](https://docs.blender.org/api/2.79b/bpy.ops.material.html) for material operators instead of `bpy.ops.mat`.